tools/c7n_gcp/c7n_gcp/resources/dns.py

```diff
@@ -55,6 +57,30 @@ def get(client, resource_info):
                         'policy': resource_info['policy_name']})
 
 
+@DnsManagedZone.filter_registry.register('records-sets')
+class DNSZoneRecordsSetsFilter(ValueFilter):
```

Comment:
This would be better as list item filter so it can better support multi attribute matches on a record.

Reply:
fixed